ingest/workflow/snakemake_rules/fetch_sequences.smk

```diff
@@ -12,6 +12,7 @@ Produces final output as
     sequences_ndjson = "data/sequences_{serotype}.ndjson"
 
 """
+workflow.global_resources.setdefault("concurrent_deploys", 2)
```

Comment:
I recommend renaming the `concurrent_deploys` resource to something that is more specific to the rule (e.g. `concurrent_fetches`) so that we don't get confused by it in the future.